src/debugger/mock-ui/UISession.js

```diff
@@ -57,8 +57,8 @@ export class UISession {
   _invalidCount: number;
   // Prepack runtime command (e.g. lib/prepack-cli.js)
   _prepackRuntime: string;
-  // Array of input source files to Prepack
-  _sourceFiles: Array<string>;
+  // Space-separated string of input source files to Prepack
```

Comment:
It's not. Using an array requires some Nuclide changes which I will put into a separate PR. For now, I've refactored the code so that as much as possible is handled as an array.